general/community/credits/thirdpartylibs.md

```diff
@@ -1003,12 +1003,12 @@ Copyright © 2003 Brent R. Matzelle (bmatzelle AT yahoo DOT com)
 
 :::note lib/markdown.php
 
-Functions to convert from the Markdown text format into clean XHTML.
+Functions to convert Markdown text format into clean XHTML.
 
-**Version**: 1.9.0 (with modifications)<br/>
+**Version**: 2.0.0 (with modifications)<br/>
```

Comment:
```suggestion
**Version**: 2.0.0<br/>
```

Reply:
I think we can remove the (with modifications) part because looking a the readme_moodle.txt there are no modifications anymore ;-)